lib/ssh.js

```diff
@@ -262,9 +263,15 @@ function _buildCommand (args) {
   return cmd.trim()
 }
 
-function _connectionDefaults() {
+function _connectionDefaults(proxyKey) {
   return {
-    port: 80
+    port: 80,
+    hostHash: 'sha256',
+    hostVerifier: function(hashedKey) {
+      var hasher = crypto.createHash('sha256');
+      hasher.update(Buffer.from(proxyKey.split(' ')[1], 'base64'));
+      return hasher.digest('hex') === hashedKey
```

Comment:
This was a bit hacky - maybe there's a cleaner way?

Reply:
It looks good to me n terms of creating this default helper function and then spreading the results. 👍